src/main/java/net/coderbot/iris/shadow/ShadowMatrices.java

```diff
@@ -23,6 +23,21 @@
 		};
 	}
 
+	public static float[] createPerspectiveMatrix(float fov) {
+		// This converts from degrees to radians.
+		float rad = (float) (1.0f / Math.tan(Math.toRadians(fov) * 0.5f));
```

Comment:
This concerts to radians, but just to call tan(). A more accurate name is yScale.

Reply:
Done